packages/drivers/odsp-driver/src/test/createNewUtilsTests.spec.ts

```diff
@@ -498,7 +486,6 @@ describe("Create New Utils Tests", () => {
 						false /* forceAccessTokenViaAuthorizationHeader */,
 						undefined /* isClpCompliantApp */,
 						true /* enableSingleRequestForShareLinkWithCreate */,
-						mockOdspResolvedUrl,
 					),
 				{
 					itemId: "mockItemId",
```

Comment:
After you have made the review changes above, let's add a test with the following flow:
- initialize OdspDriverUrlResolverForShareLink() and use it to create a resolve the request generated via createOdspCreateContainerRequest
- mock attach() api response
- then in the `container.resolvedUrl` - check if we have all the desired properties.

Checkout examples in odspCreateContainer.spec.ts.

Reply:
Since in the unit test, there is no container, you can alternatively check service. resolvedUrl. Service is returned by the createContainer method in the serviceFactory.